build-kmod-vhba.sh

```diff
@@ -12,7 +12,7 @@ if [[ "$RELEASE" -lt 39 ]]; then
   exit 0
 fi
 
-cp /tmp/ublue-os-akmods-addons/rpmbuild/SOURCES/_copr_rok-cdemu.repo /etc/yum.repos.d/
+wget "https://copr.fedorainfracloud.org/coprs/rok/cdemu/repo/fedora-${COPR_RELEASE}/rok-cdemu-fedora-${COPR_RELEASE}.repo" -O /etc/yum.repos.d/_copr_rok-cdemu.repo
```

Comment:
I'm picky, I know, but I thought we standardized on curl for the build stuff :)

Reply:
@bsherman this is pulled from another file so I'll update it in both